astropy/wcs/wcs.py

```diff
@@ -2672,18 +2675,22 @@ def footprint_to_file(self, filename='footprint.reg', color='green',
 
     @property
     def _naxis1(self):
+        warnings.warn(naxis_deprecated_message, AstropyDeprecationWarning)
```

Comment:
do we really need to deprecate this? I mean this was supposed to be private rather than in the public API, so no one should have used it

Reply:
I've used it too...there really wan't any other way to get at the information, unfortunately.

Reply:
There's very little cost to including the deprecation warning, so I'm in favor 👍